search/2016-09-01/swagger/searchservice.json

```diff
@@ -1285,7 +1288,11 @@
           "type": "string"
         },
         "name": {
```

Comment:
@anudeepsharma I have some (IMO) well-grounded objections to regex constraints. Rather than repeat them, I'll direct you to [this discussion](https://github.com/Azure/azure-rest-api-specs/pull/637) on an earlier PR.

Regarding min/max constraints, I believe they are valuable for documentation purposes where applicable. These limits are documented in the (hand written) REST API reference, which we now link to from the Swagger spec. However, I would still refrain from adding them to the Swagger until AutoRest supports [disabling client-side validation](https://github.com/Azure/autorest/issues/1583).